### ISSUE TITLE
Funny install results...

### ISSUE THREAD

**Issue by aethelwulffe** [LibreHealthIO/LibreEHR#1025]
Just did a new install.
Installer stated that it could not find or create a database, yet the installation worked.
Also, there was an apostrophe (') in the pass phrase for the admin user.  The first word was "I'm".
The reported initial password was "I" in stead of "I'm ****".

**Comment by teryhill** [comment]
yep no special characters and no spaces in the database name. I think the ' " / \ are going to act funny. the installer guys should be made aware of it.

**Comment by aethelwulffe** [comment]
Database name and pass had no special characters.  All caps though.
the initial admin user PASS had special characters.  The install worked fine.  just not the reports of the installer.  It didn't escape when displaying the passphrase when it says "Initial user "admin", passphrase "I"

**Comment by aethelwulffe** [comment]
Right...installer guys.  the project you mean.  I will take that for action on the forum.
